packages/web/src/pages/profile-page/sagas.js

```diff
@@ -436,7 +436,10 @@ function* confirmUpdateProfile(userId, metadata) {
       },
       function* () {
         yield put(profileActions.updateProfileFailed())
-      }
+      },
+      undefined,
+      undefined,
+      { operationId: 'OVERWRITE', squashable: true }
```

Comment:
eventually playlist & track updates should be able to be squashable too. will that be a problem if they are also of operationId `OVERWRITE`?

Reply:
nope will be fine since they will have different entity ids (i.e. be in different "confirm groups")! calls are grouped by `uid` first then `operationId`